### PR TITLE
ksm:update the memory size of the guest since it should be multiple of 256M on ppc

### PR DESCRIPTION
**Pull request by MiriamDeng**
ID:2101360

Signed-off-by: MiriamDeng <mdeng@redhat.com>

### CODE REVIEW

**Comment by MiriamDeng** [comment]
@PaulYuuu @maxujun can you help to review it ? Thanks.

File: qemu/tests/ksm_overcommit.py [hunk 1]
```diff
@@ -578,6 +579,9 @@ def split_parallel():
     # As we don't know the number and memory amount of VMs in advance,
     # we need to specify and create them here
     vm_name = params["main_vm"]
+    if arch.ARCH in ('ppc64', 'ppc64le'):
```
Comment (PaulYuuu) [inline qemu/tests/ksm_overcommit.py:582]: `arch.ARCH` only checks the host architecture, but you should check the guest architecture here, so use `vm_arch_name` instead.

Comment (MiriamDeng) [reply qemu/tests/ksm_overcommit.py:582]: will update it and thanks.

**Review by PaulYuuu — APPROVED** [review]

**Comment by maxujun** [comment]
LGTM，Acked.

**Pull request merged**